pom.xml

```diff
@@ -209,6 +209,12 @@
       <version>2.5</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
-0 Looks like it reduces the code that needs to be written a little bit. But I'd still be keen to keep the code lengthier, but using just vanilla JUnit.